### DISCUSSION TITLE
Info saved from ControlNet for init image used?

### DISCUSSION THREAD

**Discussion by OrganicBeej** [lllyasviel/ControlNet#505] [General]
Hi all :)

I was just a-wondering . .  if there is anywhere after an image is rendered using ControlNet with the image as DepthMap etc.. that saves the image info (title of the image) you have used for the input in ControlNet?

I sometimes have different images that look very similar which I have used as DepthMaps and it would be super useful if I could quickly see which image was used inside ControlNet for that specific render, as it's easy to get confused between 2 or 3 different versions of very similar DepthMaps/Normals etc . .

I can't see this in the terminal while rendering nor have I seen it inside 'Image Browser' extension, so just wondering if it possible as it wld be super useful for me?

Thanks
B.

**Comment by geroldmeisinger** [comment]
most Stable Diffusion UIs already store metainformation (prompt, workflow etc.) in their image generations. you can store anything you want in there, even images-in-images-in-images... but this is something which has to be implemented by the UI. you should open a feature request in the corresponding UI project. but please search first if an extension like this exists already, it sounds like it might well be.

**Reply by OrganicBeej** [reply]
Thnks for your response :) Yes, I thought it wld be something that could be easily implemented (like they have now added the chosen VAE info in metadata which is also useful).

I will have a to dble-check to see if there is an extension or another way already to edit something to make the img info from CN embed somewhere? If anyone else knows would love to hear.
Thanks again
B

**Reply by OrganicBeej** [reply]
PS: Would you know where the 'corresponding UI project' would be? Do you mean inside A1111 pages?
Thnks :)

**Reply by geroldmeisinger** [reply]
**please search first if an extension like this exists already, it sounds like it might well be.**

https://github.com/AUTOMATIC1111/stable-diffusion-webui/issues
or https://github.com/Mikubill/sd-webui-controlnet/issues

**Comment by geroldmeisinger** [comment]
duplicate https://github.com/lllyasviel/ControlNet/discussions/399
